downstream/modules/aap-on-azure/proc-azure-accessing-aap.adoc

```diff
@@ -0,0 +1,25 @@
+////
+Base the file name and the ID on the module title. For example:
+* file name: con-my-concept-module-a.adoc
+* ID: [id="con-my-concept-module-a_{context}"]
+* Title: = My concept module A
+////
+
+[id="proc-azure-accessing-aap"]
+
+= Accessing {AAPonAzureName}
+
+[role="_abstract"]
+When you initiate the deployment of the {PlatformName} managed app from Azure marketplace, a form is displayed in the *Create Red Hat Ansible Automation Platform on Microsoft Azure* window. 
+Complete the form to provision {PlatformName} infrastructure and resources into your Azure tenant.
+
+. In a web browser, navigate to *Managed Applications* in the Azure console.
+. Select the instance of {AAPonAzureName} that you deployed.
+. Select *Parameters and Outputs* in the *Settings* section in the left navigation menu.
+. Copy the URL links for {ControllerName} and {HubName} and save them. The names for the links are _automationControllerUrl_ and _automationHubUrl_.
```

Comment:
Suggestion: Add "then"
`Copy the URL links for {ControllerName} and {HubName} and save them.` > `Copy the URL links for {ControllerName} and {HubName}, and then save them.`